### PR TITLE
Update aspnet50/aspnetcore50 => dnx50/dnxcore50.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
Prepping to be merged once ready.

*Note:* We talked about still supporting aspnet50/aspnetcore50 here, should look this over in more detail.

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@NTaylorMullen__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Taylor Mullen). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

File: build.sh [hunk 1]
```diff
@@ -28,11 +28,11 @@ if test ! -d packages/KoreBuild; then
 fi
 
 if ! type k > /dev/null 2>&1; then
-    source packages/KoreBuild/build/kvm.sh
```
Comment (muratg) [inline build.sh]: "type kvm"?  @ChengTian has been making a similar change in nuget packaging.

Comment (ChengTian) [inline build.sh]: Talked to @NTaylorMullen , he will make this change in all repos except for NuGet core repos. I made this change in NuGet core repos.

**Comment by muratg** [comment]
:shipit:

**Pull request merged**